components/greaselion/browser/greaselion_service_impl.h

```diff
@@ -33,6 +35,9 @@ class ExtensionSystem;
 
 namespace greaselion {
 
+typedef std::pair<scoped_refptr<extensions::Extension>, base::ScopedTempDir>
```

Comment:
nit: in modern c++ `using` is prefered to `typedef`
also it probably can be located around `PostConvert` in  inside  the class declaration

Reply:
Old habits die hard... Fixed.